pkgs/by-name/al/alphafold3/libcifpp.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/src/alphafold3/build_data.py b/src/alphafold3/build_data.py
+index 0c33338..f3b6ab9 100644
+--- a/src/alphafold3/build_data.py
++++ b/src/alphafold3/build_data.py
+@@ -22,7 +22,7 @@ from alphafold3.constants.converters import chemical_component_sets_gen
+ def build_data():
+   """Builds intermediate data."""
+   for site_path in site.getsitepackages():
+-    path = pathlib.Path(site_path) / 'share/libcifpp/components.cif'
```

Comment:
substituteInPlace --replace-fail would do that in a more generic way. No need to maintain patches at each bump!

Reply:
This is not a core feature of the application, so there will be few updates here. Also, the intent is more explicit when patches are used, IMO.